### PR TITLE
Replace the ruby18_wrap.rb preload script with ruby_classic_wrap.rb

### LINKED ISSUE #40: Use Ancurio's Ruby v1.8 preload wrapper
When I added https://github.com/mkxp-z/mkxp-z/blob/dev/scripts/preload/ruby18_wrap.rb in #6, I was unaware that @Ancurio had already implemented the exact same (character for character) function in 2014, but with an extra function that apparently helps some games. Their version is at https://pastebin.com/w5xx0N2N (CC0 license per Matrix/Discord conversation). Probably we should nuke my version in favor of theirs.

### PR DESCRIPTION
**Pull request by WaywardHeart**
Fixes #40

### CODE REVIEW

**Comment by Splendide-Imaginarius** [comment]
Could you change the filename in the example line in `mkxp.json` accordingly?

**Comment by WaywardHeart** [comment]
Oh, right! I forgot about that.

File: scripts/preload/ruby_classic_wrap.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# ruby_classic_wrap.rb
```
Comment (Splendide-Imaginarius) [inline scripts/preload/ruby_classic_wrap.rb:1]: Could you include an Author line, similar to the old script?

File: scripts/preload/ruby_classic_wrap.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# Creative Commons CC0: To the extent possible under law, the author(s) have
```
Comment (Splendide-Imaginarius) [inline scripts/preload/ruby_classic_wrap.rb]: Also please replace `the author(s)` with whatever author name you put above.

**Comment by Splendide-Imaginarius** [comment]
pon7126 on Discord is reporting that this PR fixed an error in a VX demo they created. They're not totally sure how they triggered it but it might have had something to do with giving a character an empty skill list and then trying to list the skills.

**Comment by Splendide-Imaginarius** [comment]
@WaywardHeart just wanted to make sure you didn't miss my requested changes above.

**Comment by Splendide-Imaginarius** [comment]
(Made the changes myself, let me know if you want those changes reverted.)

**Pull request merged**